catalogue/webapp/components/WorkCard/WorkCard.js

```diff
@@ -74,6 +90,13 @@ const WorkCard = ({ work }: Props) => {
       <NextLink
         {...workUrl({
           id: work.id,
+          query,
+          page,
+          workType,
+          _queryType,
+          _dateFrom,
+          _dateTo,
+          _isFilteringBySubcategory,
```

Comment:
What's the reason for the `_` prefix?

Reply:
for `_isFilteringBySubcategory`, I added it with a view to it being temporary (it solved an immediate problem of how to do something we needed to do for one of the prototypes). The prototype it was needed for lost the battle, so this parameter can/will be killed off soon.